test/trento_web/views/v1/cluster_view_test.exs

```diff
@@ -15,17 +15,29 @@ defmodule TrentoWeb.V1.ClusterViewTest do
     end
 
     test "should remove HANA cluster V2 fields" do
+      nodes =
```

Comment:
Is this to make it explicit?
Because the factory would already create this fields, right?